cmd/kube-proxy/app/server.go

```diff
@@ -443,11 +448,18 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 	} else {
 		dbus = utildbus.New()
 		iptInterface = utiliptables.New(execer, dbus, protocol)
+		if config.Mode == proxyModeIPVS {
+			ipvsInterface = utilipvs.New(execer, dbus)
```

Comment:
Why is this a New() followed by Init() ?  Better to initialize in New(), and return (nil, error).